### PR TITLE
Add style to the app

### PR DESCRIPTION
**Pull request by zkr024**
Apply styling to the app.
Try to follow the guide but the credentials for Zeplin are not working.
image
*Use separate style sheets for the components in the app

### CODE REVIEW

File: src/components/books/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-const-assign */
```
Comment (arslanbisharat) [inline src/components/books/AddBook.js]: - [ ] Let's not disable the linter rules. These are here to help us write clean code.

File: src/components/books/style/addBook.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/* @mixin inputfield {
+  padding: 10px 14px;
+  border: 1px solid #e8e8e8;
+  border-radius: 5px;
+  outline: none;
+} */
+
```
Comment (arslanbisharat) [inline src/components/books/style/addBook.css]: - [ ] Let's remove commented code. It makes code hard to read and difficult to manage.

**Review by arslanbisharat — COMMENTED** [review]
# Changes Requested, 

Hi @zkr024, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @zkr024 👋🏻, 

Good job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**